### ISSUE TITLE
InitiateCheckout and Purchase fired twice

### ISSUE THREAD

**Issue by saimana** [facebookincubator/facebook-for-woocommerce#215]
All other events are work fine, I couldn't resolve why only InitiateCheckout and Purchase events are fired twice.

```
jQuery(function($) { 
    /* WooCommerce Facebook Integration Event Tracking */
    fbq('track', 'InitiateCheckout', {
        "source": "woocommerce",
        "version": "3.2.6",
        "pluginVersion": "1.7.4",
        "num_items": 1,
        "content_ids": "[\"wc_post_id_2420\"]",
        "content_type": "product",
        "value": "25.00",
        "currency": "BGN"
    });

    /* WooCommerce Facebook Integration Event Tracking */
    fbq('track', 'InitiateCheckout', {
        "source": "woocommerce",
        "version": "3.2.6",
        "pluginVersion": "1.7.4",
        "num_items": 1,
        "content_ids": "[\"wc_post_id_2420\"]",
        "content_type": "product",
        "value": "25.00",
        "currency": "BGN"
    });
 });
```

**Comment by tahoff** [comment]
Thanks for reporting this issue. Can you send a link to your website? Additionally, can you ensure that you're using the latest version of Facebook for WooCommerce?

**Comment by saimana** [comment]
Domain: i-tems.eu 
Plugin Version: v1.7.5

**Comment by tahoff** [comment]
I'm able to reproduce the issue on your site; however, I can't reproduce it on my test store. Are you currently using any other extensions related to checking out? If so, which ones?

**Comment by saimana** [comment]
The only one checkout integration is with getaway for direct card payments in my country: https://webops.eu/plugins/borica-plugin-for-wordpress-woocommerce/
But I have tried with a disabled plugin and get the same result in Pixel Helper.

**Comment by dmitridr** [comment]
Thank you for your patience. @tahoff Fixed this in [v1.7.7](https://github.com/facebookincubator/facebook-for-woocommerce/releases).
Please try this new version, if it's still not working, please open a new issue and reference this one. Thanks!
